vertica_python/vertica/connection.py

```diff
@@ -454,6 +454,7 @@ def _socket(self):
             raw_socket = self.enable_ssl(raw_socket, ssl_options)
 
         self.socket = raw_socket
+        self.socket_as_file = self.socket.makefile('rb', 10*1024*1024)
```

Comment:
Following the pattern, it is better to declare `self.socket_as_file = None` in `__init__()` and `reset_values()`.